app/src/main/java/com/android/solvit/shared/model/request/ServiceRequest.kt

```diff
@@ -10,17 +10,21 @@ data class ServiceRequest(
     val type: Services,
     val description: String,
     val userId: String,
+    val providerId: String? = null,
     val dueDate: Timestamp,
+    val meetingDate: Timestamp? = null,
     val location: Location?,
-    val imageUrl: String?,
-    val status: ServiceRequestStatus
+    val imageUrl: String? = null,
+    val packageId: String? = null,
+    val agreedPrice: Double? = null,
+    val status: ServiceRequestStatus = ServiceRequestStatus.PENDING,
 )
 
 enum class ServiceRequestStatus {
   PENDING,
   ACCEPTED,
-  STARTED,
-  ENDED,
+  SCHEDULED,
+  COMPLETED,
```

Comment:
I agree on this changes ! Scheduled and completed make it more official and pro !